django_perf_rec/api.py

```diff
@@ -112,7 +112,10 @@ def save_or_assert(self):
         orig_record = self.records_file.get(self.record_name, None)
 
         if orig_record is not None:
-            assert self.record == orig_record, "Performance record did not match for {}".format(self.record_name)
+            assert self.record == orig_record, "Performance record did not match for {}\n{}".format(
```

Comment:
please don't put it in the message for pytest users because pytest's output is better and has control like the `-v` flag. you can do a conditional check for pytest at the top of the file and use that when building the message, like:

```python
try:
    import pytest
    have_pytest = True
except ImportError
    have_pytest = False
```